pkg/controllers/csi/driver/volumes/host/publisher.go

```diff
@@ -49,73 +50,89 @@ type HostVolumePublisher struct {
 	path    metadata.PathResolver
 }
 
+// cyclomatic: function (*HostVolumePublisher).PublishVolume has cyclomatic complexity 13 (> max enabled 12) (revive)go-golangci-lint
+//
+//nolint:revive
```

Comment:
it's okey just to add nolint, explaining is redundunt IMO

Reply:
Done